contracts/evoting/types/ballots_test.go

```diff
@@ -382,14 +383,6 @@ func TestSubject_IsValid(t *testing.T) {
 	valid := configuration.IsValid()
 	require.True(t, valid)
 
-	// with wrongly ID not in base64
-	mainSubject.ID = "zzz"
-
-	configuration.Scaffold = []Subject{*mainSubject}
-
-	valid = configuration.IsValid()
-	require.False(t, valid)
-
```

Comment:
Not sure why the test is removed, do we not enforce base64 IDs anymore?

Reply:
Not sure anymore. Isn't it something you changed based on some errors we had on the frontend ?

Reply:
You're right, we no longer require the IDs to be base64 encoded in the configuration, but only when the ballots are cast